jsx/Form.js

```diff
@@ -228,6 +229,27 @@ class SearchableDropdown extends React.Component {
       }
     }
 
+    let newOptions = {};
+    let optionList = [];
+    if (sortByValue) {
+      for (let key in options) {
+        if (options.hasOwnProperty(key)) {
+          newOptions[options[key]] = key;
+        }
+      }
+      optionList = Object.keys(newOptions).sort().map(function(option) {
```

Comment:
this looks good. my only concern is that putting this in the render function will become costly performance-wise, but since Form elements are stateless idk if putting it in a componentDidMount method would feasible / not sure about consequences of adding state to a Form component
wondering what @HenriRabalais thinks